.changeset/yellow-dots-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/blocks-admin": minor
+---
+
+UI improvements in admin of ColumnsBlock
```

Comment:
This changeset is too generic IMO. Please change it to something like "Improve layout selection UX in `createColumnsBlock`."